pkg/ui/workspaces/db-console/src/views/app/components/errorMessage/errorBoundary.tsx

```diff
@@ -41,7 +41,9 @@ export default class ErrorBoundary extends React.Component<
 
   componentDidCatch(error: Error, errorInfo: ErrorInfo) {
     // Console.error for developer visibility.
-    console.error(error);
+    console.error("[ErrorBoundary::componentDidCatch] error = ", error);
+    console.error("[ErrorBoundary::componentDidCatch] errorInfo = ", errorInfo);
+    console.log("children = ", this.props.children);
```

Comment:
We should revert this, it was just debugging code and will cause a lot of spurious logging for end-users :)